modular_skyrat/modules/primitive_production/code/glassblowing.dm

```diff
@@ -61,6 +63,7 @@
 	desc = "A glass plate that is capable of carrying things."
 	icon = 'modular_skyrat/modules/primitive_production/icons/prim_fun.dmi'
 	icon_state = "glass_plate"
+	custom_materials = null
```

Comment:
pls add a gun here or else we can't approve the pr sorry it's just the rules! 🇺🇸

Reply:
ah my bad my bad let me add a the gun material datum, turns anything its applied to into a gun